backend/tests/legacy/test_api_allocations.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+
+
+from app.legacy.core.proposals import get_proposals_addresses
+from tests.conftest import Client, UserAccount
+
+# Please note that tests here assume that they talk to blockchain and indexer
+# whose state is not reset between tests.
+
+
+@pytest.mark.api
+def test_allocations(
+    client: Client, deployer: UserAccount, ua_alice: UserAccount, ua_bob: UserAccount
+):
+    alice_proposals = get_proposals_addresses(1)[:3]
+
+    # fund Octant
+    deployer.fund_octant(
```

Comment:
This fragment is duplicated between api tests - I think it would be good to have some helper module which will handle the setup for tests, because we will reuse it a lot